### PR TITLE
metrics: enable launch-times test on gha-run metrics script

### PR DESCRIPTION
**Pull request by dborquez**
This PR enables launch-times test on gha metrics workflow.

Fixes: #7049

### CODE REVIEW

File: tests/metrics/gha-run.sh [hunk 1]
```diff
@@ -32,25 +53,41 @@ install_kata() {
...
-	for c in kata-static-*.tar.xz; do
-		echo "untarring tarball "${c}" into ${destdir}"
-		sudo tar -xvf "${c}" -C "${destdir}"
-	done
+	sudo tar -xvf "${kata_tarball}" -C "${destdir}"
 	popd
+
+	# create symbolic links to kata components
+	for b in "${katadir}"/bin/* ; do
+		ln -sf ${b}  /usr/local/bin/$(basename $b)
```
Comment (GabyCT) [inline tests/metrics/gha-run.sh:61]: maybe `sudo` is missing here?

File: tests/metrics/time/README.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Kata Containers boot time metrics
```
Comment (GabyCT) [inline tests/metrics/time/README.md:1]: @dborquez I think that in order to add this README first we need this one https://github.com/kata-containers/kata-containers/pull/7158 to be merged in order to make a reference of this file as if we do not do it the static checks will fail

Comment (GabyCT) [reply tests/metrics/time/README.md:1]: @dborquez maybe we can submit this one later

File: tests/metrics/gha-run.sh (hunk 1 above)
Comment (dborquez) [reply tests/metrics/gha-run.sh:61]: thank you @GabyCT

**Review by GabyCT — APPROVED** [review]

**Comment by GabyCT** [comment]
@dborquez seems like there are issues with `sudo`
```
Run bash tests/metrics/gha-run.sh install-kata kata-artifacts
sudo: a terminal is required to read the password; either use the -S option to read from standard input or configure an askpass helper
sudo: a password is required
Error: Process completed with exit code 1.
```

**Comment by dborquez** [comment]
@GabyCT yes, it was required to enable **NOPASSWD** for the runner user account.
Problem fixed.

File: tests/metrics/gha-run.sh [hunk 3]
```diff
@@ -59,7 +97,7 @@ function main() {
 		install-kata) install_kata ;;
 		run-test-launchtimes-qemu) run_test_launchtimes "qemu" ;;
 		run-test-launchtimes-clh) run_test_launchtimes "clh" ;;
-		*) >&2 echo "Invalid argument"; exit 2 ;;
+		*) >&2 info "Invalid argument"; exit 2 ;;
```
[the remark this answers is not in the document]
Comment (dborquez) [reply tests/metrics/gha-run.sh:100]: The `exit 2` next to `info` causes the script to fail, the same behavior is obtained using `die`

File: tests/metrics/gha-run.sh [hunk 4]
```diff
@@ -11,46 +11,84 @@ set -o pipefail
...
+	# create symbolic links to kata components
+	for b in "${katadir}"/bin/* ; do
+		sudo ln -sf ${b}  /usr/local/bin/$(basename $b)
+	done
+
+	check_components
+	restart_containerd_service
+}
+
+check_components() {
```
[the remark this answers is not in the document]
Comment (dborquez) [reply tests/metrics/gha-run.sh:69]: sure!

File: tests/metrics/gha-run.sh [hunk 5]
```diff
@@ -11,46 +11,84 @@ set -o pipefail
...
 
 	sudo ln -sf "${source_configuration_file}" "${link_configuration_file}"
 }
 
+# Configures containerd
+overwrite_containerd_config() {
+	containerd_config_dir="/etc/containerd"
+	sudo rm -rf "${containerd_config_dir}"
+	sudo mkdir "${containerd_config_dir}"
+	sudo tee -a "${containerd_config_dir}/config.toml" << EOF
```
[the remark this answers is not in the document]
Comment (dborquez) [reply tests/metrics/gha-run.sh:33]: Agree

**Review by cmaf — APPROVED** [review]
@dborquez this lgtm but it might be good to wait for someone else's feedback for the github actions

**Review by chavafg — APPROVED** [review]

**Pull request merged**